backend/app/models/quiz.py

```
@@ -0,0 +1,16 @@
+from sqlmodel import Field, SQLModel, Relationship
```

Ruff (0.12.2)

`sqlmodel.Relationship` imported but unused

Remove unused import: `sqlmodel.Relationship`

(F401)